lib/utils/CrossfilterHistogramPlot.ts

```diff
@@ -101,6 +104,62 @@ export function CrossfilterHistogramPlot(
 				.attr("dx", (_, i) => i === 0 ? "-0.25em" : "0.25em");
 		});
 
+	//~ Background rect for the next section (hover label)
+	svg.insert("rect")
```

Comment:
Rather than adding a class, I think you could save this to a variable here:

```suggestion
	let hoverBackground = svg.insert("rect");
```

And then use the reference into the `effect` below.